com.dynamo.bob.CompileExceptionError: Failed creating AAB base.zip

**Describe the bug (REQUIRED)**
Android bundle creation fails with error "Failed creating AAB base.zip".
This occurs after adding com.helpshift:helpshift-sdkx:10.1.0 to build.gradle.
(More info: https://developers.helpshift.com/sdkx_android/getting-started/#gradle-support)

**To Reproduce (REQUIRED)**
Steps to reproduce the behavior:
1. Create a new Native Extension project.
2. Add this build.gradle to manifests/android folder:
```
dependencies {
    implementation 'com.android.support:appcompat-v7:28.0.0'
    implementation 'com.helpshift:helpshift-sdkx:10.1.0'
}
```
3. Project > Bundle > Android Application...

**Expected behavior (REQUIRED)**
No build errors

**Defold version (REQUIRED):**
  - Version 1.2.188

**Platforms (REQUIRED):**
 - Platforms: Android

**Logs (OPTIONAL):**
```
Exception in thread "main" com.dynamo.bob.CompileExceptionError: Failed creating AAB base.zip
	at com.dynamo.bob.bundle.AndroidBundler.createCompileExceptionError(AndroidBundler.java:129)
	at com.dynamo.bob.bundle.AndroidBundler.createAppBundleBaseZip(AndroidBundler.java:578)
	at com.dynamo.bob.bundle.AndroidBundler.createAAB(AndroidBundler.java:703)
	at com.dynamo.bob.bundle.AndroidBundler.bundleApplication(AndroidBundler.java:814)
	at com.dynamo.bob.Project.bundle(Project.java:729)
	at com.dynamo.bob.Project.doBuild(Project.java:1186)
	at com.dynamo.bob.Project.build(Project.java:574)
	at com.dynamo.bob.Bob.mainInternal(Bob.java:695)
	at com.dynamo.bob.Bob.main(Bob.java:744)
Caused by: java.io.IOException: Source 'extensions/helpshift/build/armv7-android/assets/helpshift' exists but is a directory
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1077)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1038)
	at com.dynamo.bob.bundle.AndroidBundler.createAppBundleBaseZip(AndroidBundler.java:525)
```
It seems like during the build process copyFile tries to copy file which is actually a directory.

--- a/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java
+++ b/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java
@@ -515,7 +515,12 @@ public class AndroidBundler implements IBundler {
             for(File asset : getExtenderAssets(project)) {
                 File dest = new File(assetsDir, asset.getName());
                 log("Copying asset " + asset + " to " + dest);
-                FileUtils.copyFile(asset, dest);
+                if (asset.isDirectory()) {
+                    FileUtils.copyDirectory(asset, dest);
+                }
+                else {
+                    FileUtils.copyFile(asset, dest);
+                }
                 BundleHelper.throwIfCanceled(canceled);
             }
 

